frontend/src/search/Searchbar.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import AsyncSelect from 'react-select/async';
+import { SearchResultEntry } from './SearchResultEntry';
+import useService from '../dependency-injection/useService';
+import SearchService from '../services/searchService';
+import convertSearchResultToEntries from './SearchEntryConverter';
+
+export default function Searchbar(): JSX.Element {
+  const customStyles = {
+    option: (provided: any) => ({
+      ...provided,
+      color: 'black',
+    }),
+  };
+
+  const searchService = useService(SearchService, null);
+
+  const loadOptions = function (
+    inputValue: string,
+    callback: (entries: SearchResultEntry[]) => void
+  ) {
+    searchService
+      .fullTextSearch(inputValue)
+      .then((result) => callback(convertSearchResultToEntries(result)));
+    // TODO Error handling for bad requests
+  };
+
+  /* eslint-disable react/jsx-no-bind */
```

Comment:
Add a comment why you disabled that.
`/* eslint-disable react/jsx-no-bind -- my cool comment */`